client/extensions/woocommerce/app/order/order-fulfillment/index.js

```diff
@@ -106,8 +131,61 @@ class OrderFulfillment extends Component {
 		}
 	}
 
+	renderFulfillmentAction() {
+		const { labelsLoaded, order, site, translate, hasLabelsPaymentMethod } = this.props;
+		const isShippable = this.isShippable( order );
+		const renderLabelsButton = wcsEnabled && hasLabelsPaymentMethod;
+
+		if ( ! renderLabelsButton && ! isShippable ) {
+			return null;
+		}
+
+		if ( ! renderLabelsButton ) {
+			return (
+				<Button primary onClick={ this.toggleDialog }>{ translate( 'Fulfill' ) }</Button>
+			);
+		}
+
+		const onLabelPrint = () => this.props.openPrintingFlow( order.id, site.ID );
+		const buttonClassName = classNames( {
+			'is-placeholder': ! labelsLoaded,
+		} );
+
+		if ( ! isShippable ) {
```

Comment:
I'm confused - why would we want to show a Print new label button if an order is not shippable? Is this for a Print the Label Again flow?